### ISSUE TITLE
you don't need this

### ISSUE THREAD

**Issue by Dmitry-N-Medvedev** [BigShanJohn/registration-service#1]
https://github.com/BigShanJohn/registration-service/blob/6a52007e7fd0012d6bf50ccba173da5d7ad6ef3d/server/server.js#L1

like I emphasized during the interview: you are not expected to write any HTTP servers

**Comment by BigShanJohn** [comment]
Okay.   should I use WebSocket approach? since you mentioned it as part of what you are using.

**Comment by Dmitry-N-Medvedev** [comment]
@BigShanJohn , no. You don't need any servers to implement the microservice in question.

**Comment by BigShanJohn** [comment]
ok. Got it
